packages/account/package.json

```diff
@@ -33,7 +33,7 @@
         "@deriv-com/analytics": "1.10.0",
         "@deriv-com/translations": "1.3.4",
         "@deriv-com/utils": "^0.0.25",
-        "@deriv-com/ui": "1.29.9",
+        "@deriv-com/ui": "1.29.10",
```

Comment:
Updated to incorporate fix for `Text` component typescript error: https://github.com/deriv-com/ui/pull/232